floip/__init__.py

```diff
@@ -27,14 +31,15 @@ def xform_from_floip_dict(survey, name, values):
     values - the floip question object with the type, label and question
              options for the question.
     """
-    question_type = QUESTION_TYPES[values['type']]
+    question_type = (QUESTION_TYPES[values['type']]
+                     if values['type'] in QUESTION_TYPES else values['type'])
```

Comment:
Should we have all the question types in `QUESTION_TYPES`,  which means this change will be unnecessary. It will allow someone who imports and looks at `floip.QUESTION_TYPES` knows all supported question types.